dbt/include/firebolt/macros/catalog.sql

```diff
@@ -3,28 +3,29 @@ the columns (for instance, `is_nullable` is missing) but more could be added lat
 
 {% macro firebolt__get_catalog(information_schemas, schemas) -%}
   {%- call statement('catalog', fetch_result=True) %}
+     SELECT *, ROW_NUMBER() OVER ( PARTITION BY table_name ORDER BY table_name ASC ) as column_index FROM (
```

Comment:
What is a column_index in this case? Do I understand correctly that this value is just artificial?
Do

Reply:
Yeah, this is similar to snowflake's ordinal position - https://github.com/dbt-labs/dbt-snowflake/blob/main/dbt/include/snowflake/macros/catalog.sql#L47  or postgres `attnum`
Basically it denotes the order in which columns are shown. We don't really have this exposed in Firebolt, but `ROW_NUMBER` allows us to simulate the behaviour. This allows us to conform to the result dbt expects from the catalog query.